index.html

```diff
@@ -1855,73 +1855,73 @@ <h2>Advanced Concepts</h2>
 this section explores more complex topics about <a>verifiable credentials</a>.
       </p>
 
-      <section class="informative">
```

Comment:
Start of `section` tag was deleted, but closing tag wasn't, resulting in mismatched section tags throughout the document. Please fix mismatched tags.

Reply:
On second thought, I don't understand why this section was deleted and merged into the "Advanced Concepts" section, as it has more to do w/ lifecycle than advanced concepts. My guess is that this was an accidental deletion? In that case, this PR should probably put the section header back in.

Reply:
Accidental deletion. Restoring the section header.